### PR TITLE
Feat/theme choices into Develop

### PR DESCRIPTION
**Pull request by pieryvdl**
So in this pull request I made wholesale changes to the themes. 
Note: Do not evaluate it as a complete product, I tried to model the component to match the one on figma, but there are small things that are not right. (Like the icon, because I could not get a gaming controller icon). Also note the colors are not completely correct, I used the colors @shan-226 used on Figma. @u22520016 you are welcome to change them if you like. 
You'll find that: 
1. Themes are now named "light_green" or "dark_green" when saved to Firebase. Please change your theme colors on Firebase for the code to work. (If merged in, we will change all the themes.) 
2. When selecting a color the color should change to the one you selected, light and dark mode will also change depending on the switch setting (for light and dark mode). 
3. Also note if we change the theme primary colors, we should update the component colors as well. 

Please review this from a functionality standpoint and remember to change back the theme to light/dark when running and older branch, until this is merged into Develop. All feedback welcome. 😄

### CODE REVIEW

**Comment by MonikaTheiss** [comment]
Hi @pieryvdl 
When I tried to run it, I got this error:
image
, every other page loads but the customize page does not load

**Comment by MonikaTheiss** [comment]
Ok it works on my side now , forgot about the api key. It works really cool, I just saw a little error , I believe it happens when you save the changes and then quickly go back to the profile page.  Although now that I try to recreate the error it works fine, I did however get that error twice at the start..

here is the error:
image

**Review by u22520016 — APPROVED** [review]
Thank you @pieryvdl !  I think it looks and works great!  We can just adjust the colours in the future but for now the functionality is there and it works. Good job😁

File: lib/view/pages/settings/customize_page.dart [hunk 1]
```diff
@@ -47,11 +50,13 @@ class CustomizeProfilePageObject extends State<CustomizeProfilePage> {
       var response = await http.get(url);
       if (response.statusCode == 200) {
         var decoded = json.decode(response.body);
-        setState(() {
-          _genres = (decoded['results'] as List)
-              .map((genre) => genre['name'].toString())
-              .toList();
-        });
+        if (_isMounted) {
```
Comment (MonikaTheiss) [inline lib/view/pages/settings/customize_page.dart:53]: I see what you did here, thats pretty smart !

**Review by MonikaTheiss — APPROVED** [review]
Code looks good and it looks good on my phone. Thank you for looking into that error I found! That's a very cool solution you found!

**Pull request merged**